src/libmongoc/tests/json/initial_dns_seedlist_discovery/load-balanced/loadBalanced-no-hosts.json

```diff
@@ -0,0 +1,8 @@
+{
+    "uri": "mongodb+srv://test4.test.build.10gen.cc/?loadBalanced=true",
+    "seeds": [],
+    "hosts": [],
+    "error": true,
+    "comment": "Should fail because loadBalanced is true but the SRV record resolves to 0 hosts"
```

Comment:
Noting that this is pulled from https://github.com/mongodb/specifications/pull/1035